tests/foreman/sys/test_hot_backup.py

```diff
@@ -0,0 +1,309 @@
+        present.
+
+        """
+        with _get_connection() as connection:
+            dir_name = make_random_tmp_directory(connection)
+            result = connection.run(
+                'katello-backup /tmp/{0} --online-backup '
+                '--skip-pulp'.format(dir_name),
+                output_format='plain'
+            )
+            self.assertEqual(result.return_code, 0)
+            self.assertIn(BCK_MSG.format(dir_name), ' '.join(result.stdout))
+            files = connection.run(
+                    'ls /tmp/{0}/katello-backup'.format(dir_name),
+                    'list'
+                    )
+            self.assertNotIn(u'pulp_data.tar', files.stdout)
+
+    @tier3
+    @skip_if_bug_open('bugzilla', 1399244)
+    @skip_if_bug_open('bugzilla', 1384901)
+    def test_positive_incremental(self):
+        """Make an incremental backup
+
+        @id: a5b3536f-8365-41c7-9386-cddde588fe8c
+
+        @Steps:
+
+        1. Run full backup (base)
+        2. Make config change c1
+        3. Run incremental backup ib1
+        4. Restore base backup, verify c1 config doesnt not exist
+        5. restore ib1, verify c1 config does exist
+
+        @Assert: Backup "ib1" is backed up.
+
+        """
+        with _get_connection() as connection:
+            b1_dir = make_random_tmp_directory(connection)
+            # run full backup
+            result = connection.run(
+                'katello-backup /tmp/{0} --online-backup'.format(b1_dir),
+                output_format='plain'
+            )
+            self.assertEqual(result.return_code, 0)
+            self.assertIn(BCK_MSG.format(b1_dir), ' '.join(result.stdout))
+            files = connection.run(
+                    'ls /tmp/{0}/katello-backup*'.format(b1_dir),
+                    'list'
+                    )
+            # backup could have more files than the default so it is superset
+            self.assertTrue(set(files.stdout).issuperset(set(BACKUP_FILES)))
+
+            # Create a new repo
+            repo_name = gen_string('alpha')
+            entities.Repository(
+                    product=self.product, name=repo_name
+            ).create()
+
+            # run incremental backup /tmp/ib1
+            ib1_dir = gen_string('alpha')
+            connection.run('cp -r /tmp/{0} /tmp/{1}'.format(b1_dir, ib1_dir))
+            timestamped_dir = connection.run(
+                    'ls /tmp/{0}/'.format(ib1_dir),
+                    'list'
+                    )
+            result = connection.run(
+                'katello-backup --online-backup --incremental /tmp/{0}/{1}'
+                .format(ib1_dir, timestamped_dir.stdout[0]),
+                output_format='plain'
+            )
+            self.assertEqual(result.return_code, 0)
+            self.assertIn(BCK_MSG.format(ib1_dir), ' '.join(result.stdout))
+
+            # restore /tmp/b1 and assert repo 1 is not there
+            connection.run('katello-restore /tmp/{0}'.format(b1_dir))
```

Comment:
@abalakh @pondrejk 
When this test runs, `kattelo-restore` will restore and then restart the system. So it may fail the concurrent tests.
#3905 and #4088  must be resolved